packages/typeorm/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@next-auth/typeorm-adapter",
+  "repository": "https://github.com/nextauthjs/adapters",
+  "version": "0.0.0",
+  "description": "TypeORM adapter for next-auth.",
+  "author": "Iain Collins",
+  "license": "ISC",
+  "keywords": [
+    "next-auth",
+    "next.js",
+    "oauth",
+    "typeorm"
+  ],
+  "homepage": "https://next-auth.js.org",
+  "bugs": {
+    "url": "https://github.com/nextauthjs/adapters/issues"
+  },
+  "main": "index.js",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "test": "yarn run db:start && yarn run test:db && yarn run db:stop",
+    "test:db": "yarn run test:db:mysql && yarn run test:db:postgres && yarn run test:db:mongodb && yarn run test:db:mssql",
+    "test:db:mysql": "./tests/mysql.sh",
+    "test:db:postgres": "./tests/postgres.sh",
+    "test:db:mongodb": "node tests/mongodb.js",
+    "test:db:mssql": "node tests/mssql.js",
+    "db:start": "docker-compose -f tests/docker/databases.yml up -d",
+    "db:stop": "docker-compose -f tests/docker/databases.yml down --remove",
+    "lint": "standard",
+    "lint:fix": "standard --fix"
+  },
+  "dependencies": {
+    "crypto-js": "^4.0.0",
+    "require_optional": "^1.0.1",
+    "typeorm": "^0.2.30"
+  },
+  "devDependencies": {
+    "assert": "^2.0.0",
+    "jest": "^26.6.3",
+    "mongodb": "^3.6.3",
+    "mssql": "^6.3.1",
+    "mysql": "^2.18.1",
+    "next-auth": "^3.1.0",
+    "pg": "^8.5.1",
+    "standard": "^16.0.3",
+    "typeorm-adapter": "^1.2.0"
+  },
+  "peerOptionalDependencies": {
+    "next-auth": "^3.1.0",
```

Comment:
I think `next-auth` should be under `peerDependencies`

Reply:
Yeah exactly, I didnt change any of the internal typeorm stuff. That was all taken 1 to 1 from the original adapter code. 

I'll add a readme and update the peer dependency stuff now and then yeah once we have this repo organised and looking good we can focus on taking the adapters out of the primary repo :+1: